packages/web-forms/src/components/common/map/useMapInteractions.ts

```diff
@@ -76,11 +108,50 @@ export function useMapInteractions(mapInstance: Map): UseMapInteractions {
 		}
 	};
 
+	const onSelectFeatureOrVertex = (
+		event: MapBrowserEvent,
+		onSelect?: (feature: Feature | undefined, selectedVertexIndex: number | undefined) => void
+	): void => {
```

Comment:
Unless there are sideeffects this call does nothing if the `onSelect` function is not given, right? If so, we can shortcircuit all the work, and also remove the `?` throughout.

```suggestion
	): void => {
	  if (!onSelect) {
	  	return;
	  }
```